File: abhipro/abhiprojects/switch.cpp
#include<iostream>
using namespace std;
int main()
{

	int x=10;
	int y=20;
	int z=0;

	cout<<"x="<<x<<endl;
	cout<<"y="<<y<<endl;

	char ch;
        ch=2;


	switch(ch)
	{
		case 1: z=x+y;
         		cout<<"addition of x+y is:"<<z<<endl;
	        	break;

		case 2: z=x-y;
			cout<<"substraction of x-y is"<<z<<endl;
			break;

		case 3: z=x%y;
                        cout<<"division of x-y is"<<z<<endl;
                        break;

		case 4: z=x*y;
                        cout<<"multiplication of x-y is"<<z<<endl;
                        break;
		default :
			cout<<"thank you"<<endl;

	}
	return 0;
}

File: abhipro/abhiprojects/Switch.java
class Switch
{
	public static void main(String args[])
	{
		int a=10;
		int b=20;
		int c=0;

		char ch;
		ch=3;

		System.out.println("A="+ a);
	 	System.out.println("B="+ b);
		System.out.println("C="+c);

		switch(ch)
		{
			case 1:
				c=a+b;
			 	System.out.println("Addition="+ c);
				break;
			case 2:
                                c=a-b;
                                System.out.println("Subtraction="+ c);
				break;
			case 3:
                                c=a*b;
                                System.out.println("Multiplication="+ c);
				break;
			case 4:
                                c=a/b;
                                System.out.println("Division="+ c);
				break;
			default :
				System.out.println("Thank you");
		}
	}
}
